DOCS-3373: Document 3DS2 for cardholder data payments (#2419)
MultiSafepay/docs: content/payment-methods/cardholder-data.md

Set `type` to `direct`.
<br>

## 3DS2

> When you collect cardholder data, you are also responsible for collecting the contextual information (fingerprint) of the customer's device required for [3DS2](/docs/3ds2) authentication. The fingerprint can be created through JavaScript interfaces and methods in the customer's browser.

Reviewer: ```suggestion
When you collect cardholder data, you must also collect the contextual information about the customer's device (fingerprint) required for [3DS2](/docs/3ds2) authentication. The fingerprint can be created through JavaScript interfaces and methods in the customer's browser.

```
